sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2226,16 +2226,18 @@ class Dataset[T] private[sql](
     * `column`'s expression must only refer to attributes supplied by this Dataset. It is an
     * error to add a column that refers to some other Dataset.
     *
-    * You can choose to add new columns either at the end (default behavior) or at the beginning.
+    * The position of the new column start from 0, and a negative position means at the end (default behavior).
```

Comment:
"starts at `0`. Any negative position means to add the column at the end"?

Reply:
I modified as you suggested